experiments/template_SLTC.py

```diff
@@ -122,6 +130,11 @@ def main():
     set_seed(training_args.seed)
 
     train_dataset, eval_dataset, predict_dataset = get_data(training_args, data_args)
+    train_dataset = train_dataset.filter(lambda example: len(example['input']) > 0)
+    eval_dataset = eval_dataset.filter(lambda example: len(example['input']) > 0)
+    predict_dataset = predict_dataset.filter(lambda example: len(example['input']) > 0)
+
+    print(train_dataset['label'])
```

Comment:
is this a debug output?

Reply:
Not really neccesary, but I thougt it's nice to have it in the output file. I can remove it.

Reply:
No it's fine. But please use the logger instead of print and add some more information

Reply:
ok :)

Reply:
thanks :)